recipes/mastercard/runner.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+source $(pwd)/bin/config.sh
+
+BASEDIR=$(dirname $0)
+NAME=$(basename $BASEDIR)
+VERSION=$DATE
+
+
+(      
+    cd $BASEDIR
+    mkdir -p input
+    mkdir -p output
+    chmod 777 input
+    
+    #For testing purposes
+    #cp Geogrids_NYC_Zip_Code_Level_Jan2019_Feb2021.zip input/
+
+    #will download all files from mastercard. Then mastercard will delete after successfull download.
+    scp -v -P 22022  -i /root/.ssh/id_rsa_axway -o "StrictHostKeyChecking=no" newyorkcity@files.mastercard.com:geoinsights/data/fromMC/* /input
+    echo 'file is ' $(find $BASEDIR/input -name "*.zip" ) 
+    unzip -d /input -P $MASTERCARD_PASSWORD $(find $BASEDIR/input -name "*.zip" | head -1 ) 
```

Comment:
@SteveScott is there a way to implement a check that there is only one file? what is MC uploads multiple files?